### PR TITLE
Prepare v3 pre-release

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Update version for pre-release 3.0.0-next.202401041628

## Release Notes

Milestone: vNext

Changelog: n/a

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by zFernand0 — DISMISSED** [review]
LGTM! 😋 
Should we remove the Theia workflow form the `next` branch?

**Comment by JillieBeanSim** [comment]
I created an issue other day for removing Theia checks and accompanying code 
https://github.com/zowe/vscode-extension-for-zowe/issues/2647

**Review by zFernand0 — APPROVED** [review]

**Comment by zFernand0** [comment]
> I created an issue other day for removing Theia checks and accompanying code #2647

Thanks for removing the workflow. And yes, we can target all other code changes around dropping Theia support in that issue 😋

**Comment by JillieBeanSim** [comment]
I see an issue on `next` branch with the Issue Unix command with following error via USS treeview. Will investigate
Screenshot 2024-01-08 at 9 36 06 AM

**Comment by zFernand0** [comment]
> I see an issue on `next` branch with the Issue Unix command with following error via USS treeview. Will investigate Screenshot 2024-01-08 at 9 36 06 AM

hey @JillieBeanSim,
I noted this behavior in the following issue:
- https://github.com/zowe/vscode-extension-for-zowe/issues/2643

I didn't see it as a showstopper though

**Comment by zFernand0** [comment]
> also this during startup of extension Screenshot 2024-01-08 at 9 38 33 AM

oh, and this one might be related to...
- https://github.com/zowe/vscode-extension-for-zowe/issues/2577

**Comment by JillieBeanSim** [comment]
> > also this during startup of extension Screenshot 2024-01-08 at 9 38 33 AM
> 
> oh, and this one might be related to...
> 
> * [Startup for Zowe Explorer and ZE extensions can be slow #2577](https://github.com/zowe/vscode-extension-for-zowe/issues/2577)

I think something went wrong with merge from main for the command error on start up since this isn't experienced with main.
The other is breaking since the Unix command isn't issued and console isn't opened, I just recieve the error about username.  
Looking into it

**Comment by JillieBeanSim** [comment]
the command was in the "view/item/context" array but not in the "command" array of package.json where they are initially created and also not registered during activation.  It also doesn't exist on main branch, so removing.

**Comment by zFernand0** [comment]
Hey @JillieBeanSim,
Thanks for looking into this and fixing it! 🥳 

---

Regarding:
>I think something went wrong with merge from main

@traeok, @t1m0thyj, do you think that fewer of these cases will occur if the improvements to the deployment workflow (that you guys brought up) were rolled out?

File: packages/zowe-explorer/package.json [hunk 1]
```diff
@@ -1405,11 +1405,6 @@
           "command": "zowe.jobs.editSession",
           "group": "099_zowe_jobsProfileModification@0"
         },
-        {
-          "when": "view == zowe.jobs.explorer && viewItem =~ /^(?!.*_fav.*)server.*/",
-          "command": "zowe.jobs.removeJobsSession",
```
Comment (t1m0thyj) [inline packages/zowe-explorer/package.json:1410]: Looks like this menu item was referencing a non-existent command, so makes sense to remove it 👍

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @JillieBeanSim!

**Comment by traeok** [comment]
> @traeok, @t1m0thyj, do you think that fewer of these cases will occur if the improvements to the deployment workflow (that you guys brought up) were rolled out?

Yep, this would minimize the amount of commits on merge PRs since `next` would be based on `main` once v3 is GA

**Review by traeok — APPROVED** [review]
LGTM, thanks @JillieBeanSim!

**Comment by JillieBeanSim** [comment]
Thanks @t1m0thyj & @traeok for reviewing so quickly

**Pull request merged**